app/tests/test_solution_service.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from app import solution_service
+
+
+class SolutionServiceTests(unittest.TestCase):
```

Comment:
Nice, can see this one getting massive. 
Might be worth investigating parameterising these testes within an list of like (QuestionStr, AnswerStr, ExpectedResult) so we can run through a whole heap of cases without having to write new cases for them.

Reply:
great idea